index.html

```diff
@@ -4,72 +4,133 @@
   <main id="main-content">
 
     <section class="p-strip--image is-dark" style="background-image: url('https://assets.ubuntu.com/v1/8bc629a7-microk8s-suru-background.svg'); background-size: cover;">
-      <div class="row u-equal-height">
+      <div class="row u-vertically-center u-equal-height">
         <div class="col-8">
-          <h1>Kubernetes for workstations and appliances</h1>
-          <p class="p-heading--four">A single package of k8s that installs on <s>41</s> 42 flavours of Linux. Made for developers and great for appliances.</p>
+          <h1>
+            Zero-ops Kubernetes for workstations and edge / IoT
+          </h1>
+          <p class="p-heading--four">
+            A single package of k8s for that installs 42 flavours of Linux. Made for developers, and great for edge, IoT and appliances.
+          </p>
+        </div>
+        <div class="col-4 u-align--center u-hide--small">
+          <img src="https://assets.ubuntu.com/v1/96d35aa4-certified-kubernetes-logo.svg" width="150" alt="Certified Kubernetes" />
+        </div>
+      </div>
+    </section>
+
+    <section class="p-strip is-shallow">
+      <div class="row p-divider">
+        <div class="col-4 p-divider__block u-align--center">
+          <h5>Linux</h5>
           <a href="https://snapcraft.io/microk8s" title="Get it from the Snap Store">
             <img src="https://snapcraft.io/static/images/badges/en/snap-store-white.svg" alt="Get it from the Snap Store" width="200" />
           </a>
         </div>
-        <div class="col-4 prefix-1 u-vertically-center">
-          <img src="https://assets.ubuntu.com/v1/96d35aa4-certified-kubernetes-logo.svg" alt="Certified Kubernetes" />
+        <div class="col-4 p-divider__block u-align--center">
+          <h5>Windows</h5>
+          <p>
+            <a href="https://multipass.run/">
+            <img src="https://assets.ubuntu.com/v1/ea34f006-Multipass+logomark_rgb.svg" width="62" />
+            <br />
+            Run it with Multipass
+            </a>
+          </p>
+        </div>
+        <div class="col-4 p-divider__block u-align--center">
+          <h5>macOS</h5>
+          <p>
+            <a href="https://multipass.run/">
+            <img src="https://assets.ubuntu.com/v1/ea34f006-Multipass+logomark_rgb.svg" width="62" />
+            <br />
+            Run it with Multipass
+            </a>
+          </p>
         </div>
       </div>
     </section>
 
-    <section class="p-strip is-shallow is-bordered">
-      <div class="row">
-        <h2 class="p-muted-heading">Features</h2>
+    <section class="p-strip--light">
+      <div class="row u-vertically-center">
+        <div class="col-6">
+          <blockquote  class="p-pull-quote">
```

Comment:
Could you try use the Vanilla classes: https://docs.vanillaframework.io/patterns/pull-quote/